jasmine/jasmine.d.ts

```diff
@@ -382,7 +382,7 @@ declare module jasmine {
         /** By chaining the spy with and.throwError, all calls to the spy will throw the specified value. */
         throwError(msg: string): void;
         /** When a calling strategy is used for a spy, the original stubbing behavior can be returned at any time with and.stub. */
-        stub(): Spy;
+        stub(...params:any[]): Spy;

```

Comment:
https://github.com/jasmine/jasmine/blob/4acb7448d866698128eaa9bb5c89d027ac24f434/lib/jasmine-core/jasmine.js#L1932
I think stub method is not consume arguments. is this right?